opm/autodiff/SimulatorBase_impl.hpp

```diff
@@ -109,7 +110,60 @@ namespace Opm
             adaptiveTimeStepping.reset( new AdaptiveTimeStepping( param_, terminal_output_ ) );
         }
 
-        output_writer_.writeInit();
+
```

Comment:
I have now read the code below, and it looks correct. I still think that this is something the output layer should handle (since it is necessary only because of the very peculiar "logical cartesianish" layout of the output format), but that would require the output layer to know about the grid (the real grid, not just the input specification used), or at least the topology (connections) of it. So for now I'm willing to include this.